ui-guidelines/index.html

```diff
@@ -300,14 +304,399 @@ <h5 class="header-black top-extra">SPECIFICATIONS</h5>
+								</div>
+								<div class="weight-sample">
+									<p class="big-letter normal">A</p>
+									<p class="regular u-move-up-xlarge">MEDIUM (500)</p>
+								</div>
+								<div class="weight-sample">
+									<p class="big-letter medium">A</p>
+									<p class="regular u-move-up-xlarge">SEMI-BOLD (600)</p>
+								</div>
+								<div class="weight-sample">
+									<p class="big-letter bold">A</p>
+									<p class="regular u-move-up-xlarge">BOLD (700)</p>
+								</div>
+							</div>	
+							<h4 class="u-move-down-xlarge">Type Sizes</h4>
+							<p class="regular">We use different type sizes and weights to define a layout composition.</p>
+							<div class="typography u-move-left-xlarge u-space-top">
+								<h1 class="text-h1">eZ is an open source content management system</h1>
+								<p class="u-move-up-xsmall regular"><code class="label-code">h1</code><span class="text-size">2.5em</span></p>
+								<h2 class="text-h2">eZ is an open source content management system</h2>
+								<p class="u-move-up-xsmall regular"><code class="label-code">h2</code><span class="text-size">1.5em</span></p>
+								<h3 class="text-h3">eZ is an open source content management system</h3>
+								<p class="u-move-up-xsmall regular"><code class="label-code">h3</code><span class="text-size">1.25em</span></p>
+								<p class="u-move-down text-x-huge">eZ is an open source content management system</p>
+								<p class="u-move-up regular"><code class="label-code">.text-x-huge</code><span class="text-size">2.5em</span></p>
+								<p class="u-move-down text-huge">eZ is an open source content management system</p>
+								<p class="u-move-up-small regular"><code class="label-code">.text-huge</code><span class="text-size">1.875em</span></p>
+								<p class="u-move-down text-xx-large">eZ is an open source content management system</p>
+								<p class="u-move-up-xsmall regular"><code class="label-code">.text-xx-large</code><span class="text-size">1.4em</span></p>
+								<p class="u-move-down-large text-x-large">eZ is an open source content management system</p>
+								<p class="u-move-up-xsmall regular"><code class="label-code">.text-x-large</code><span class="text-size">1.133em</span></p>
+								<p class="u-move-down-large text-large">eZ is an open source content management system</p>
+								<p class="u-move-up-xsmall regular"><code class="label-code">.text-large</code><span class="text-size">1.1em</span></p>
+								<p class="u-move-down-large text-xx-medium">eZ is an open source content management system</p>
+								<p class="u-move-up-xsmall regular"><code class="label-code">.text-xx-medium</code><span class="text-size">1.066em</span></p>
+								<p class="u-move-down-large text-x-medium">eZ is an open source content management system</p>
+								<p class="u-move-up-xsmall regular"><code class="label-code">.text-x-medium</code><span class="text-size">1em</span></p>
```

Comment:
given how the style are supposed to work, I think we should use `rem` instead of `em`. Otherwise as soon as we start using a class other than `text-x-medium`, the font size won't be what you expect for instance, if you have something like:

```
<p class="text-xx-large">Large text <span class="text-x-medium">and normal</span></p>
```
The font-size in the `span` element will still be the same as the rest of the text in the paragraph.